### ISSUE TITLE
"make lakectl public" goreleaser stage of release process is broken

### ISSUE THREAD

**Issue by arielshaqed** [treeverse/lakeFS#2893]
Cannot release 0.58.0 :-(

See e.g. https://github.com/treeverse/lakeFS/runs/4953014184?check_suite_focus=true

**Comment by arielshaqed** [comment]
```
Run aws s3 cp --recursive --acl public-read "s3://treeverse-clients-us-east/lakectl/$(echo ${GITHUB_REF##*/} | cut -d. -f1-2)" "s3://treeverse-clients-us-east/lakectl/$(echo ${GITHUB_REF##*/} | cut -d. -f1-2)" --metadata-directive REPLACE
  aws s3 cp --recursive --acl public-read "s3://treeverse-clients-us-east/lakectl/$(echo ${GITHUB_REF##*/} | cut -d. -f1-2)" "s3://treeverse-clients-us-east/lakectl/$(echo ${GITHUB_REF##*/} | cut -d. -f1-2)" --metadata-directive REPLACE
  shell: /usr/bin/bash -e {0}
  env:
    GOROOT: /opt/hostedtoolcache/go/1.16.2/x64

<botocore.awsrequest.AWSRequest object at 0x7f1002968ac0>
Error: Process completed with exit code 255.
```

**Comment by arielshaqed** [comment]
Tried to re-run the goreleaser action, that doesn't work because it already managed the previous state and refuses to do it again.  https://github.com/treeverse/lakeFS/runs/4953299115?check_suite_focus=true

**Comment by arielshaqed** [comment]
I think we introduced this step recently, in #2885.

**Comment by johnnyaug** [comment]
Solved in #2944
